game/views.py

```
@@ -1154,109 +1154,59 @@ def password_reset_account_selection(request):
-                    request,
-                    'Failed to send confirmation email.'
-                )
-
-            return redirect('index')
+If this wasn't you, ignore this email.""",
+                from_email=settings.EMAIL_HOST_USER,
+                recipient_list=[user.email],
+                fail_silently=False,
+            )
+            messages.success(request, 'Confirmation email sent to your registered email.')
+        except Exception:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)